fbchat/_file.py

```diff
@@ -155,6 +155,18 @@ def _from_graphql(cls, data):
             uid=data.get("legacy_attachment_id"),
         )
 
+    @classmethod
+    def _from_list(cls, data):
+        data = data["node"]
+        return cls(
+            width=data["original_dimensions"].get("x"),
+            height=data["original_dimensions"].get("y"),
+            thumbnail_url=data["image"].get("uri"),
+            large_preview=data["image2"],
```

Comment:
Do the previews have a known, constant size? If so, we should provide the sizes here

Reply:
![image](https://user-images.githubusercontent.com/42300497/61626265-1ed9f200-ac7d-11e9-9e3f-cb50a16ebd82.png)
Response looks like this so I don't a lot of items to work with.
`image` is a thumbnail (no size given),
`image1` is a medium preview (size given, parsed by \_\_init\_\_),
`image2` is a original **(?)** file (size given, parsed by \_\_init\_\_).

Reply:
The same is is for video btw.

Reply:
Whoops, I forgot about the parsing in `__init__` 👍